rpi-setup.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+echo "ARE YOU SURE YOU WANT TO DELETE ANY CONFIGURATION"
+echo "EXISTING AND SET UP A NEW STARTUP SERVICE?"
+echo ""
+echo "IF NOT, PRESS <CTRL+C> TO ABORT"
+echo ""
+
+systemd="/etc/systemd/system/signalk.service"
+dir=$(pwd)
+
+
+echo -n "Enter your vessel name and press [enter]:"
+read vesselName
+
+UUIDFile="$dir/UUID"
+if [ -f $UUIDFile ]
+then
+  UUID=$( cat $UUIDFile)
+  echo "UUID=$UUID"
+else
+  UUID=$( cat /proc/sys/kernel/random/uuid)
+  sudo touch $UUIDFile
+  sudo echo $UUID > $UUIDFile
+  echo "UUID generated: $UUID"
+fi
+
+
+vesselBash="$dir/bin/$vesselName"
+vesselJson="$dir/settings/$vesselName.json"
+
+echo "A file will be created with your settings in"
+echo "$vesselJson."
+echo "This uses stored NMEA data to set up the server,"
+echo "but settings can be changed by going to <ipaddress>:3000,"
+echo "selecting \"Server Plugins Configuration\""
+echo "and \"Vessel Setup\""
+
+sudo touch $vesselBash
+sudo echo "#!/bin/sh" > $vesselBash
+sudo echo "" >> $vesselBash
+sudo echo "DIR=\`dirname \$0\`" >> $vesselBash
+sudo echo "\${DIR}/signalk-server -s /settings/$vesselName.json \$*" >> $vesselBash
+sudo chmod 775 $vesselBash
+
```

Comment:
How about using here documents instead of a multitude of echo lines? http://www.tldp.org/LDP/abs/html/here-docs.html